### ISSUE TITLE
wesnoth.game_config api change

### ISSUE THREAD

**Issue by sevu** [wesnoth/wesnoth#5178]
Items below `wesnoth.game_config.mp_settings.`* are different between 1.14 and 1.15.

Doesn't affect me, just noticed it might breaking compatibility for someone.

**Comment by Pentarctagon** [comment]
I would guess it's caused by 0efd35acbcba45724be4737073dc7f77480ec781.

**Comment by gfgtdf** [comment]
Its casued by exposing an internal datastructure without thinking...

In any case we should redesign that api anyways see https://github.com/wesnoth/wesnoth/issues/4907

**Comment by Pentarctagon** [comment]
Should this be closed as won't fix then, or is it worth leaving open?

**Comment by CelticMinstrel** [comment]
I would say some compatibility path should be added regardless of what happens with #4907. However, it's also probably best to wait on that until after #4907 is resolved, as the form of the compatibility path will depend on what's done there.

**Comment by Pentarctagon** [comment]
Given how much time has passed, is it useful to keep this open?

**Comment by CelticMinstrel** [comment]
I'd lean towards still doing it, but it's true that the usefulness declines as more time passes.

**Comment by sevu** [comment]
Probably not
